### DISCUSSION TITLE
[solved] Bluetooth Connection only on second try

### DISCUSSION THREAD

**Discussion by Demel75** [SBFspot/SBFspot#601] [General]
Everything works fine upto bluettoth first connection.

Connecting to 00:80:XX:XX:22:8X (1/10)
Connecting to 00:80:XX:XX:22:8X (2/10)
Initialising...

On Second try all data will be read.

Anyone else this issue?

**Comment by sbf-** [comment]
> On Second try all data will be read

So not really an issue...
Do you have a reliable connection? BT_signal might give an idea.

**Reply by Demel75** [reply]
Connecting to 00:80:XX:XX:22:8X (1/10)
Connecting to 00:80:XX:XX:22:8X (2/10)
Initialising...
SUSyID: 125 - SessionID: 966127905
SMA netID=06
Serial Nr: 7EF8XXXX (2130223XXX)
BT Signal=76.5%

There are only 2 meters between BT dongle and SMA

**Reply by sbf-** [reply]
How many inverters do you have?

**Comment by Demel75** [comment]
Only one

**Comment by sbf-** [comment]
Not sure if this will help but to my knowledge NetID should be 1 when connecting to only 1 inverter.
https://manuals.sma.de/SB30-50TL-21/en-US/391165451.html

**Comment by Demel75** [comment]
I never set this. I try to set ist to one. Thanks for advise.

**Reply by Demel75** [reply]
No change to behaviour of bluetooth connect. Everytime on second try it got connect.

**Reply by Demel75** [reply]
I think I got it. The bluetooth dongle was working on USB EHCI Port. I changed to USB XHCI and I got no dupllicates at the moment.
Strange.
Thanx all for assist. Discussion could be closed.
